pipelines/build/regeneration/jdk10_regeneration_pipeline.groovy

```diff
@@ -12,16 +13,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-String javaVersion = "jdk10u"
+String javaVersion = "jdk10"
 
 node ("master") {
   try {
     def scmVars = checkout scm
     load "${WORKSPACE}/pipelines/build/common/import_lib.groovy"
   
-    def buildConfigurations = load "${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy"
+    // Load buildConfigurations from config file. This is what the nightlies & releases use to setup their downstream jobs
+    def buildConfigurations = null
+    def buildConfigPath = "${WORKSPACE}/pipelines/jobs/configurations/${javaVersion}_pipeline_config.groovy"
+    try {
+      buildConfigurations = load buildConfigPath
+    } catch (NoSuchFileException e) {
```

Comment:
The issue is that we will almost always have a try catch instance as `load` is actively searching for a filepath and will throw an exception if it can't find it. We could use [fileExists](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#fileexists-verify-if-file-exists-in-workspace) but doesn't that just add the same code, just longer? I'm open to other ideas though if you have one in mind

Reply:
I'd have a preference for using the exception at the end if we can't find either file, but if that gets too convoluted then let me know.

Reply:
Ok I worked around it. Could I get a re-review @karianna (Test run: https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/alpha_pipeline_jobs_generator/6/console)